crates/storage/src/collections/vec/fuzz_tests.rs

```diff
@@ -0,0 +1,84 @@
+// Copyright 2018-2021 Parity Technologies (UK) Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use super::Vec as StorageVec;
+use crate::{
+    test_utils::FuzzCollection,
+    traits::{
+        KeyPtr,
+        PackedLayout,
+        SpreadLayout,
+    },
+    Pack,
+};
+
+use quickcheck::{
+    Arbitrary,
+    Gen,
+};
+use std::vec::Vec;
+
+impl<T: Arbitrary + PackedLayout + Send + Clone + 'static> Arbitrary for StorageVec<T>
```

Comment:
please move these trait bounds into the `where` clause as done in the rest of the crate.